src/cloud_tasks/cli.py

```
@@ -626,252 +470,933 @@ async def list_running_instances_cmd(args: argparse.Namespace, config: Config) -
+        db_file = f"{provider_config.job_id}.db"
+
+    logger.info(f"Using database file: {db_file}")
+
+    # Initialize variables that will be used in finally block
+    task_db = None
+    event_monitor = None
+
+    try:
+        if args.continue_run:
+            # CONTINUE MODE: Resume from previous run
+            logger.info(f"Continuing job '{provider_config.job_id}' from database '{db_file}'")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)